### PR TITLE
Awesome book

### PR DESCRIPTION
**Pull request by Eternalgratis**
- Collection that keeps a list of books.
- Function to add a new book to the collection, with title and author.
- Function to remove a book from the collection.
- Display all books saved in localStorage and in the collection.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
```
Comment (Kingstalux) [inline script.js]: - [ ] Good job so far 👍 but in this project, you are required to use JavaScript objects and arrays and not classes as you will be refactoring your code in subsequent projects to use classes. So kindly remove all classes and use simple functions and objects to achieve your objective 💯 
- Requirements:
Screenshot 2022-07-05 125521

File: script.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+document.querySelector('#book-form').addEventListener('submit', (e) => {
+  // Prevent actual submit
+  e.preventDefault();
+
+  // Get form values
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+
+  // Instatiate book
+  const book = new Book(title, author);
```
Comment (Kingstalux) [inline script.js]: - [ ] (OPTIONAL) It would be great to give a random `id` to each book so that you can easily delete a book using that `id` 👍

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Eternalgratis and @aron-helu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

175083599-879bf673-fb93-406f-939c-de7e586bf6d9

## Highlights 👏 
✅ App has the correct functionality 🔥 
✅ Linters are passing 👍 
✅ Good commit history 👏 


## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly edit your README file using this [Microverse template](https://github.com/microverseinc/readme-template) with all partner's info and info about how a user can set up and run your project locally 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**